src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/util/EntityParserTest.java

```diff
@@ -49,8 +57,7 @@ public void should_parse_envelop_data_with_no_non_scannable_items_in() throws IO
 
     @Test
     public void should_parse_envelop_data_with_no_scannable_items_in() throws IOException {
```

Comment:
envelop -> envelope

Reply:
since I am moving that file to relevant place (due to getting rid of `EntityParser`), I will rename those in next pr so not conflicting atm as is. same can be said about strange metafile name